graph/src/data/store/scalar.rs

```diff
@@ -19,7 +19,15 @@ use std::str::FromStr;
 
 pub use num_bigint::Sign as BigIntSign;
 
-/// All operations on `BigDecimal` return a normalized value.
+/// These are the limits of IEEE-754 decimal128, a format we may want to switch to. See
+/// https://en.wikipedia.org/wiki/Decimal128_floating-point_format.
+pub const BIG_DECIMAL_MIN_EXP: i32 = -6143;
```

Comment:
This is nit-picky, but these can be made associated constants and not pollute the already large prelude namespace:
```rust
impl BigDecimal {
    pub const MIN_EXP: i32 = -6143;
}

fn test() {
   let exp = BigDecimal::MIN_EXP;
}
```

Reply:
That really is nicer!